### DISCUSSION TITLE
Import of a markdown file in another markdown file

### DISCUSSION THREAD

**Discussion by rmrbytes** [nuxt/content#2025] [Q&A]
I am developing a blog site and all content is in markdown format. For the same I am using the following structure.

```
- content
     index.md
     about.md
     include.md
- pages
  [...slug].vue
```

The slug.vue is as follows:
```
<template>
    <div>
         <ContentDoc/>
    </div>
</template>
```

The md files are as follows:
```
# Index Page

This is the home page

```

In the About Page I would like to include another md file say "include.md"

```
# About Page

Here I wish to import include.md but not sure how to do it.

@import './include.md' did not work
```

Is this possible in Nuxt3? thanks

**Comment by nobkd** [comment]
If you just want to render the `include.md` in the `about.md`, you can use the [`<ContentDoc>`](https://content.nuxtjs.org/api/components/content-doc) component in the `about.md` file.

You can do this by using it as tag or with [MDC syntax](https://content.nuxtjs.org/guide/writing/mdc#inline-components).

```md
<!-- /about.md -->

:content-doc{path="/include"}

or

<ContentDoc path="/include" />
```

Please be aware, that circular dependencies can break your page.

**Reply by rmrbytes** [reply]
When I try the above using Nuxt 3, I get an error

```
Document not found, overwrite this content with #not-found slot in <ContentDoc>.
```

All the examples in the MDC link seem to suggest to import a vue component inside an md file
